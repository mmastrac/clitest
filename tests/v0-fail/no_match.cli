#!/usr/bin/env clitest --v0

$ printf "hello\n"
! bye
