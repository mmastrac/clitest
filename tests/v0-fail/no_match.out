Running <root>/no_match.cli ...

printf "hello\n"
---
hello
---
ERROR: pattern literal at line <root>/no_match.cli:4 does not match output line "hello"
matching Sequence([Literal(bye), End])
  matching Literal(bye)
  literal FAILED match: "hello" == "bye"

‚ùå FAIL

<root>/no_match.cli FAILED
Error: pattern literal at line <root>/no_match.cli:4 does not match output line "hello"

1/1 test(s) failed
